Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery/Definitions/Result.cs

```diff
@@ -9,6 +11,25 @@ public class Result
         /// Boolean value of success of the query 
         /// </summary>
         /// <example>true</example>
-        public bool Success { get; protected set; }
+        public bool Success { get; private set; }
+
+        /// <summary>
+        /// String value of the Error message that comes from Oracle.
+        /// </summary>
+        /// <example>"ORA-01722: invalid number"</example>
+        public string Message { get; private set; }
+
+        /// <summary>
+        /// List of JObjects showing the output of the query.
+        /// </summary>
+        /// <example>[{"ID": "1","FIRST_NAME": "Saija","LAST_NAME": "Saijalainen","START_DATE": ""}]</example>
+        public JToken Output { get; private set; }
```

Comment:
Have you verified that this works in Frends? At least I have noticed that dot notation doesn't work with the result object if you return JToken as a property. If you change property type to `dynamic` or `object` then you should be able to use dot notation.

Reply:
It didn't, changed the result.Output to be dynamic